parlai/core/torch_agent.py

```diff
@@ -351,6 +361,11 @@ def __init__(self, opt, shared=None):
         self.history = deque(maxlen=self.histsz)
         # truncate == 0 might give funny behavior
         self.truncate = opt['truncate'] if opt['truncate'] >= 0 else None
+        self.left_truncate = (self.truncate if not opt.get('left_truncate')
```

Comment:
Could do `self.left_truncate = self.get('left_truncate', self.truncate)`.

Maybe we should add a guard about negative numbers.

Reply:
oh duh, oops